geometry_msgs/Pose goal
---
bool result
string message